digital-twins/jsonld-contexts/observed_attributes.jsonld

```diff
@@ -0,0 +1,13 @@
+{
+  "@context": {
+    "UseCase": "https://smartdatamodels/UseCase",
+    "UseCaseConfig": "https://smartdatamodels/UseCaseConfig",
+    "name": "https://smartdatamodels.org/name",
```

Comment:
better use https://schema.org/name (standard and we already use in some other contexts)